stencil/src/components/event-session-registration-modal/event-session-registration-modal.scss

```diff
@@ -0,0 +1,216 @@
+:host {
+    display: block;
+}
+
+.event-session-modal-container {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    background-color: black;
+    z-index: 9999;
+    top: 0;
+    left: 0;
+
+    .hidden {
+        display: none;
+    }
+
+    .modal-container {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 100%;
+        height: 100%;
+
+        h2, h3, h4, h5, h6, p {
+            color: black;
+        }
+
+        .modal {
+            height: 85%;
+            width: 85%;
+            background-color: white;
+            border-radius: 5px;
+            padding: 1.5rem;
+            overflow-y: scroll;
+
+            .title-container {
+                display: flex;
+                border-bottom: 1px solid black;
+
+                .title-text {
+                    width: 75%;
+                    text-align: left;
+                    h2 {
+                        color: black;
+                    }
+                }
+
+                .close-icon {
+                    width: 25%;
+                    text-align: right;
+
+                    i {
+                        cursor: pointer;
+                        font-size: 1.5rem;
+                    }
+                }
+            }
+
+            .session-select-container {
+                text-align: left;
+
+                p {
+                    font-size: 1.25rem;
+                    margin-top: 2rem;
+                }
+            }
+
+            .session-select-list {
+                list-style: none;
+                text-align: left;
+                display: flex;
+                flex-wrap: wrap;
+                align-items: stretch;
+
+                .checkbox-container {
+                    width: 10%;
+                    position: relative;
+                    cursor: pointer;
+
+                    input {
+                        position: absolute;
+                        opacity: 0;
+                        height: 0;
+                        width: 0;
+                    }
+
+                    .checkmark {
+                        position: absolute;
+                        height: 25px;
```

Comment:
I'm noticing a few flips to px instead of rem here; I'd think of checkmarks as measured based on text size as they should scale on the text size if someone flips to large print (elsewhere, heights are either in rem or percent). Is that intentional?